Presentation.Web/app/components/it-project/tabs/it-project-tab-communication.controller.ts

```diff
@@ -21,7 +21,7 @@
                             //get the role names
                             return $http.get("odata/ItProjectRoles?%24format=json&%24top=100&%24orderby=priority+desc&%24count=true")
                                 .then(function (roleResult) {
-                                    var roles: { Name }[] = roleResult.data.value;
```

Comment:
den manglende "Id" property skal tilføjes her. Det er en type spec. Prøv det i stedet for dynamic type